Atomic/util.py

```diff
@@ -40,9 +40,6 @@ def gomtree_available():
 def runc_available():
     return os.path.exists(RUNC_PATH)
 
-def bwrap_oci_available():
```

Comment:
we don't use it anywhere now.

bwrap-oci should not be required for user images, in the same way as we don't require runc for system containers.